onnxruntime/test/contrib_ops/skiplayernorm_op_test.cc

```diff
@@ -27,18 +27,32 @@ static void RunTest(
     bool no_beta = false,
     bool simplified = false,
     bool use_token_count = false,
-    bool strict = false) {
+    bool strict = false,
+    bool batch_size_1 = false,
+    bool no_batch_size = false) {
   // Input and output shapes
   //   Input 0 - input: (batch_size, sequence_length, hidden_size) or (batch_size * sequence_length, hidden_size)
-  //   Input 1 - skip : (batch_size, sequence_length, hidden_size) or (batch_size * sequence_length, hidden_size)
+  //   Input 1 - skip : (batch_size, sequence_length, hidden_size) or (batch_size * sequence_length, hidden_size) or (1, sequence_lemgth, hidden_size)
   //   Input 2 - gamma: (hidden_size)
   //   Input 3 - beta : (hidden_size)
   //   Output         : (batch_size, sequence_length, hidden_size) or (batch_size * sequence_length, hidden_size)
   std::vector<int64_t> input_dims = {batch_size, sequence_length, hidden_size};
+  std::vector<int64_t> skip_dims = input_dims;
+
+
+
   if (use_token_count) {
     input_dims = {batch_size * sequence_length, hidden_size};
   }
-  std::vector<int64_t> skip_dims = input_dims;
+
+  if(batch_size_1 == true){
+    skip_dims = {1, sequence_length, hidden_size};
+  }
+
+  if(no_batch_size){
+    skip_dims = {sequence_length, hidden_size};
```

Comment:
Do we support this format? It is not in the comment of line 35.

Reply:
My apologies, I added some new edits to support that format.